### DISCUSSION TITLE
Can mods be used to summarize a YT video?

### DISCUSSION THREAD

**Discussion by vladimircezar** [charmbracelet/mods#247] [Q&A]
Is this possible?

**Comment by maaslalani** [comment]
Hey! I don't believe this is currently possible with _just_ using `mods`. But if there were a command line tool or service that could retrieve the transcript or transcribe a (YouTube) video then you could use mods to summarize the transcript. For example:

```bash
transcribe https://www.youtube.com/watch?v=dQw4w9WgXcQ | mods "Summarize this transcript in 6 sentences"
```

**Reply by maaslalani** [reply]
This may be worth looking into as a transcription service: https://github.com/ArthurFDLR/whisper-youtube
